source/extensions/filters/http/dynamo/dynamo_request_parser.h

```diff
@@ -58,6 +58,11 @@ class RequestParser {
    */
   static TableDescriptor parseTable(const std::string& operation, const Json::Object& json_data);
 
+  /**
+   * @return string name of table in transaction object, or empty string if none
+   */
+  static std::string getTableNameFromTransactItem(const Json::Object& transact_item);
```

Comment:
you could return an optional rather than an empty string.

Reply:
ok!